index.js

```diff
@@ -13,14 +14,16 @@ const {
   additionalQueryParams,
   labelsToAdd,
   message,
-  dryRun=true // Safer to force you to turn it on
+  numDaysOld = 0,
+  dryRun = true // Safer to force you to turn it on
 } = require('./config.json');
 
 const assignmentMessage = (message, assignee) => {
   return message.replace(new RegExp(/@@/, 'g'), '@' + assignee);
```

Comment:
I didn't notice this before I guess, but you don't need either the `{}` or the `return` here